tests/Windows/WindowTest.php

```diff
@@ -1,13 +1,16 @@
 <?php
 
+use Native\Laravel\Client\Client;
 use Native\Laravel\Facades\Window;
 use Native\Laravel\Windows\Window as WindowClass;
 
 it('test window', function () {
+    Http::fake();
     Window::shouldReceive('open')
         ->andReturn(new WindowClass('main'));
 
     $window = Window::open()
+        ->setClient(new Client)
```

Comment:
the `title()` method in `Window.php` calls the Client internally, which isn't set at that point

Reply:
This is fine for now. We can work out the dependencies for better testing later